vendor/models/offer.py

```diff
@@ -206,6 +206,9 @@ def get_next_billing_date(self):
             start_date = self.term_start_date
         return get_payment_scheduled_end_date(self, start_date)
 
+    def get_subscription_start_date(self, profile):
```

Comment:
you are sending the profile but its is not declared in the invoice function defeition

Reply:
sorry my bad got confused with some many repeated names function defenitions